constants/rateLimtingMiddelware.js

```diff
@@ -0,0 +1,23 @@
+const RATE_LIMITER_SLOW_BRUTE_BY_IP_OPTIONS = {
+  points: 100,
+  duration: 60 * 60 * 24,
+  blockDuration: 60 * 60 * 24,
+  keyPrefix: "login_fail_by_ip_per_day",
+};
+
+const RATE_LIMITER_FAST_BRUTE_BY_IP_OPTIONS = {
+  keyPrefix: "login_fail_by_ip_per_minute",
+  points: 5,
+  duration: 30,
+  blockDuration: 60 * 10,
+};
```

Comment:
How did we arrive at these figures. Also should these numbers be same for user and super user. 🤔

Reply:
I just picked from the official wiki of this rate limiter package, for starting.
Thanks for pointing that out, I didn't think about this.

> I will add limits as per different roles.